### DISCUSSION TITLE
Seattle Motor Accidents

### DISCUSSION THREAD

**Discussion by derrickn-1720724** [jakobzhao/geog495#65] [Final Project]
Application: https://derrickn-1720724.github.io/Geog495SeattleMotorAccidents/
Github: https://github.com/derrickn-1720724/Geog495SeattleMotorAccidents

Description: The project is intended for individuals to see the difference in trends between months, dates, and years on motor accidents. A big factor on what we are able to do with this is able to see specific days and see how much more accidents are prone on these days. For example, New Years Eve and New Years day. With these options, we are able to see a very in-depth analysis of trends over time. My goal through this project is to allow others to be much more knowledgeable on what days are more prone to accidents and to help them be much more cautious. We are able to use Mapbox and several different web services such as github to further showcase our project.
![model](https://user-images.githubusercontent.com/32498758/145534521-104a401f-9a2e-40a0-acad-a7b762c99732.jpg)

**Comment by LiamGearyUW** [comment]
There's very little I could criticize about this map, it's simply very effective at getting information across while looking clean. What few improvements I could think of making would likely require tools that we haven't yet learned of. Perhaps the injuries section could use a change: gradients are useful when working with large numbers or decimals, but when we're dealing solely with integers (and a small number at that), perhaps a regular legend of "this color is 0, this color is 1, this color is 2, etc." would be better at avoiding confusion.

**Comment by seanhy1** [comment]
This was an amazing project! It is quite interesting how you and I decided to use the same exact data, template, and filtered inputs. The only difference between ours is the fact that I decided to filter days based on weekday/weekend or all. The only suggestion I have for you is to keep the design of your side-panel consistent. In other words, you decided to add a ':' after the year and month slider inputs but chose to not include a semicolon for the day slider input or the text label for the injuries legend. Besides that though, I don't see much I would personally change to this application. Well done!

**Comment by Zhiyuan-Jiao** [comment]
Hello. I think this is a really cool map with a lot of filter function and sliders. It is amazing how those circle change size based on the number of injuries. One thing that could be improved from my perspective is that to change the street color and make the street name more stand out from the rest of the map so that the user could see which street have more car accident happened more easily.
